### ISSUE TITLE
Example missing for alert() / confirm()

### ISSUE THREAD

**Issue by erapoport** [lumapps/lumX#364]
In the docs site under Services >  Notifications, there is a demo for `alert()` and `confirm()` but doesn't have an example of how to use it.

**Comment by JuanSomniis** [comment]
Well alert is simple


```javascript
LxNotificationService.alert('text');
```
And confirm 

```javascript
LxNotificationService.confirm('text?', { cancel:'Cancelar', ok:'Aceptar' },
      function(answer){
        if (answer) {
        //ok answer
        }else {
       //cancel answer
        }

      });
```

**Comment by matmkian** [comment]
We've just released LumX v1.0.0rc1. You now have a real documentation ;-)
